src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -100,19 +106,23 @@ private ResponseEntity postEventInternal(final String eventTypeName,
         }
     }
 
+    private void reportSLOs(final long startingNanos, final int totalSizeBytes, final int eventCount) {
+        final long msSpent = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startingNanos);
+        LOG.info("[SLO] [publishing-latency] time={} size={} count={}", msSpent, totalSizeBytes, eventCount);
```

Comment:
Shouldn't we report only successfully published events? or somehow indicate that not all of them were published in slo log entry ..